R/generics.R

```diff
@@ -318,6 +318,22 @@ setMethod("obj_na_str<-", "ANY", function(obj, value) {
   obj
 })
 
+#' @rdname lab_name
+#' @export
+setGeneric("obj_align", function(obj) standardGeneric("obj_align"))
+#' @rdname lab_name
+#' @exportMethod obj_align
+setMethod("obj_align", "ANY", function(obj) attr(obj, "align", exact = TRUE))
```

Comment:
This shouldn't be using attr, these are S4 classes, so it should be `@`, need to get all the way out of the habit of accessing slots with attr.

Reply:
Thanks @gmbecker , we can rasise a new issue for the refactoring here.

Reply:
Done in https://github.com/insightsengineering/formatters/pull/179